packages/material-ui-lab/src/TreeView/TreeView.js

```diff
@@ -98,31 +104,136 @@ const TreeView = React.forwardRef(function TreeView(props, ref) {
    */
 
   const getNextNode = (id) => {
```

Comment:
This is depth-first, right?

Reply:
Kind of yes, except it, doesn't go the full tree depth. But it moves in a depth search like manner.